src/senaite/core/catalog/indexer/configure.zcml

```diff
@@ -67,4 +67,8 @@
   <!-- Worksheet Indexer -->
   <adapter name="listing_searchable_text" factory=".worksheet.listing_searchable_text"/>
 
+  <!-- AnalysisProfile Indexes -->
+  <adapter name="sampletype_title" factory=".analysisprofile.sampletype_title"/>
+  <adapter name="sampletype_uid" factory=".analysisprofile.sampletype_uid"/>
```

Comment:
We need an upgrade step to add these indexes to the Setup Catalog

Reply:
These indexes are already present in setup catalog

Reply:
No need, cause the field `sample_types` was not there before this Pull Request